src/middlewared/middlewared/plugins/ldap.py

```diff
@@ -155,9 +155,15 @@ def _open(self):
             saved_simple_error = None
             saved_gssapi_error = None
             for server in self.hosts:
+                parts = server.split(':', 1)
```

Comment:
@anodos325 

```
>>> 'foo:'.split(':')
['foo', '']
```

which means port will be empty string, not sure we care enough and/or want to validate it?